src/tests/nssdb.rs

```diff
@@ -102,6 +102,142 @@ fn test_nssdb_token() {
     testtokn.finalize();
 }
 
+#[test]
+#[parallel]
+#[cfg(feature = "mldsa")]
+fn test_nssdb_token_mldsa() {
+    let name = String::from("test_nssdb_token_mldsa");
+    let datadir = "testdata/nssdbdir2";
```

Comment:
would be good to have the issue described elsewhere than in the source code. Either simple README.md in the nssdb directory or just few lines saying what is the problem we are testing for with this particular DB. But otherwise I do not have any other concerns.
The CI failure is the flaky test. I will rerun.